doc/mkdocs/docs/api/macros/json_has_three_way_comparison.md

```diff
@@ -0,0 +1,24 @@
+# JSON_HAS_THREE_WAY_COMPARISON
+
+```teratermmacro
+#define JSON_HAS_THREE_WAY_COMPARISON /* value */
+```
+
+Wether the compiler and standard library support 3-way comparison.
+Possible values are `1` when supported or `0` when unsupported.
+
+## Default definition
+
+Unless defined by the user prior to inclusion of the `json.hpp` header, the macro definition is roughly equivalent to the follwing:
```

Comment:
See above - do not give a complete definition here - the interested reader will like the code more anyway. :)